db/fixtures/paths/foundations/seed.rb

```diff
@@ -101,7 +101,9 @@
   section.identifier_uuid = 'd2476929-d71b-4b25-969b-8f7da6a40c94'
 
   section.add_lessons(
-    foundation_lessons.fetch('CSS Foundations'),
+    foundation_lessons.fetch('Intro to HTML and CSS'),
```

Comment:
Same here please, RE: this lesson comes earlier in the course.

```suggestion
```

Reply:
Got it